one/src/lib.rs

```diff
@@ -239,6 +242,27 @@ impl Network {
     }
 }
 
+#[derive(Args, Debug)]
+struct KeyOpts {
+    /// Path to private key directory
```

Comment:
can we clarify somehow *what* private key we are talking about here?

I'm assuming this is the key that is used by libp2p and is what the PeerID derives from?

Eventually we may also additional private keys such as:
- a node DID private key for the node to sign messages and identify itself on the network (ideally this would re-use the same key as libp2p is using for the peerid I think)
- an ethereum wallet private key
- an adm chain wallet private key

So we should be specific here about what we're talking about